### DISCUSSION TITLE
채팅 말고도 귓속말도 TTS가 읽던데 혹시 의도하신건가요?

### DISCUSSION THREAD

**Discussion by playCosmos** [Lastorder-DC/chatreader-kor#57] [문의] [answered]
제목 그대로 귓속말도 TTS가 읽습니다.
스트리머 계정이 아니라 다른 계정으로 접속하라고 한 이유가 이것인 듯 한데
귓속말이랑 채팅이랑 구별해서 읽지 않거나 귓속말 읽기 옵션을 주셔야 할 것 같습니다.

**Comment by Lastorder-DC** [comment]
의도하지 않은 현상이며 발생한다면 버그입니다.

**Comment by Lastorder-DC** [comment, marked as answer]
수정되었습니다.
